### PR TITLE
refactor: 将条件格式迷你柱形图绘制及主题配置统一归并到mini图中

### PR DESCRIPTION
**Pull request by xingwanying**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [ ] Code style optimization
- [x] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

image

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

**Comment by lcx-seima** [comment]
g5.0 那个分支 rebase 难受了。。

File: packages/s2-core/src/cell/data-cell.ts [hunk 1]
```diff
@@ -53,10 +50,18 @@ export class DataCell extends BaseCell<ViewMeta> {
 
   protected conditionIconShape: GuiIcon;
 
+  public get cellOnditions() {
```
Comment (wjgogogo) [inline packages/s2-core/src/cell/data-cell.ts:53]:
```suggestion
  public get cellConditions() {
```

**Review by lcx-seima — APPROVED** [review]

**Pull request merged**